transaction.go

```diff
@@ -12,23 +12,30 @@ type snowflakeTx struct {
 	ctx context.Context
 }
 
+type txCommand int
+
+const (
+	commit txCommand = iota
+	rollback
+)
+
+func (cmd txCommand) string() string {
```

Comment:
I think that it would be safer to explicitly bind the enum to the string representation like:
```go
func (cmd txCommand) string() (txStr string, err error) {
	switch cmd {
	case commit:
		return "COMMIT", nil
	case rollback:
		return "ROLLBACK", nil
	}
	return "", errors.New("incorrect tx command")
}
```

Reply:
I double checked and current codebase convention is to use string arrays. Let's discuss this later and create new issue to refactor enums if needed.

Reply:
Added suggestion.